README.md

```diff
@@ -0,0 +1,229 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Vet Clinic Database] <a name="about-project"></a>
+
+> In this project, The Blog app will be a classic example of a blog website. I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+- [x] Database👌
+- [x] ROR (Ruby on Rails) 
+- [x] Postgresql
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+
+- [ ] Create a new app. Give it a meaningful name.
+- [ ] Initialize my project with Git.💯
+- [ ] Build my project schema.
+      Create and run the necessary migration files.
+      - Table and column names match the ERD diagram.
+      - note: photo for users table  be a link to an image
+      - Foreign keys be included.
+      - All columns that are foreign keys have a corresponding index.👌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+> Comming soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a pc and install ant editor you prefer💯
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd Blog-App
+  git clone https://github.com/osamaashraf6/Blog-App
+```
+
+
+### Install
+
+Install this project with:
+
+
+
+
+```sh
+  cd my-project
+  gem install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+```sh
+  rails server
+```
+
+
+
+### Deployment
+
+You can deploy this project using: Render or Netlify
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
```

Comment:
- [ ] If this **Deployment** sub-section is not useful for you then you can remove it to make your readme file to the point and clean :+1: